build.cake

```diff
@@ -9,7 +10,7 @@ var trigger = Argument("trigger", "");
 
 var nugetSource = "https://api.nuget.org/v3/index.json";
 var solutionFileName = "Faithlife.Utility.sln";
-var docsAssembly = $@"src\Faithlife.Utility\bin\{configuration}\net46\Faithlife.Utility.dll";
+var docsAssembly = File("src/Faithlife.Utility/bin/" + configuration + "/net46/Faithlife.Utility.dll").ToString();
```

Comment:
Would `string.Format` be easier to read, here and below? I guess I'm not sure...